combat-trainer.lic

```diff
@@ -1599,10 +1622,9 @@ class SpellProcess
                    .select { |spell| spell['day'] ? UserVars.sun['day'] : true }
                    .select { |spell| spell['slivers'] ? DRSpells.slivers : true }
                    .select { |spell| spell['starlight_threshold'] ? enough_starlight?(game_state, spell) : true }
+
     data = if @offensive_spell_cycle.empty?
-             skills = ready_spells.map { |spell| spell['skill'] }
-             skill = game_state.sort_by_rate_then_rank(skills).first
-             ready_spells.find { |spell| spell['skill'] == skill }
+             ready_spells.min_by { |spell| DRSkill.getxp(spell['skill']) }
```

Comment:
⚡ Merge conflict, this is inadvertently reverting changes introduced in https://github.com/rpherbig/dr-scripts/pull/4686 that should be kept